OasisDEX/oasis-borrow
# 🧿 Ajna empty pages for open borrow POC

# 🧿 Ajna empty pages for open borrow POC

Quick and dirty token pair selector for Ajna and empty page for opening flow.

Planned urls examples that are going to work with proposed structure:
`/ajna/borrow/ETH-DAI`
`/ajna/earn/wBTC-USDC`
`/ajna/multiply/ETH-stETH`
At this moment only `borrow` is added to the list of products and tokens list is completely random, but that shows the idea.
  
## Changes 👷‍♀️

- Created selector page,
- created empty open page,
- moved `productCardsAjna` from `consts.ts` to `content.ts` as this is file is more suitable for other things.
  
## How to test 🧪

- Go to `/ajna/borrow`,
- select random tokens couple of times,
- check if empty page that you can go to after selection is valid.
